store/etcd/v3/etcd.go

```diff
@@ -423,8 +426,7 @@ func (s *EtcdV3) NewLock(key string, options *store.LockOptions) (lock store.Loc
 
 	go func() {
 		<-renewCh
-		session.Close()
-		return
+		_ = session.Close()
```

Comment:
Same here, why assign?

Reply:
if you want to ignore an error, it's better to do it explicitly.